src/test/java/land/oras/OCILayoutTest.java

```diff
@@ -1037,6 +1156,7 @@ private void assertLayerExists(Path ociLayoutPath, Layer layer) {
     }
 
     private void assertBlobExists(Path ociLayoutPath, String digest) {
+        String digestWithPrefix = digest.startsWith("sha256:") ? digest : "sha256:" + digest;
```

Comment:
Even if only test please use SupportedAlgorithm.fromDigest(digest).getPrefix() instead of hardcoding prefix